### PR TITLE
Fix: Tunnel --PORT parameter not working in Node.js app.

### PR DESCRIPTION
**Pull request by paul-phan**
### WHY are these changes introduced?
After version 2.7.0, we no longer can use a custom port for `ngrok` tunnel. Seem like there was a missing parameter in the `node_service.rb`

### WHAT is this pull request doing?
Add the `port` parameter to the tunnel start function.

### How to test your changes?
For Node.js project, assume you want to tunnel the `localhost:3000`. 
This command should work: `shopify app serve --PORT 3000`

### Update checklist

- [x] I've added a CHANGELOG entry for this PR (if the change is public-facing)
- [x] I've considered possible cross-platform impacts (Mac, Linux, Windows).
- [x] I've left the version number as is (we'll handle incrementing this when releasing).
- [ ] I've included any post-release steps in the section above.

### CODE REVIEW

File: lib/shopify_cli/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ShopifyCLI
-  VERSION = "2.7.0"
+  VERSION = "2.7.1"
```
[the remark this answers is not in the document]
Comment (paul-phan) [reply lib/shopify_cli/version.rb]: Hi @gonzaloriestra, thanks for reviewing!
I've updated the changelog file and tried to add tests as well.
I'm not entirely familiar with Ruby code, so please let me know if you need any further updates.

**Review by gonzaloriestra — COMMENTED** [review]
Thanks for the changes! I've already tested it and it works as expected 👌 

There are just a few minor details to be fixed and we can merge it!

**Comment by paul-phan** [comment]
Thanks for your suggestion; I've updated it!

**Review by gonzaloriestra — APPROVED** [review]
Ready to merge 🚀 

It will be included in the next release. Thanks!

**Pull request merged**